PRODUCTION_ENVIRONMENT_INSTALLATION.md

```diff
@@ -0,0 +1,323 @@
+# Production environment installation
+
+## Prerequisites
+
+1. We have `root` access to new VPS server with Ubuntu 16.04 preinstalled.
+1. Proper domain (`dev.volontulo.pl`, `rc.volontulo.pl` or `volontulo.pl`) was redirected into new VPS server IP address.
```

Comment:
VPS's server IP

Reply:
I'm leaving here `VPS's IP` as `VPS` stands for Virtual Private Server